front/lib/api/llm/index.ts

```diff
@@ -56,5 +58,20 @@ export async function getLLM(
     });
   }
 
+  if (isFireworksWhitelistedModelId(modelId)) {
+    const { FIREWORKS_API_KEY } = dustManagedCredentials();
+    if (!FIREWORKS_API_KEY) {
+      throw new Error("FIREWORKS_API_KEY environment variable is required");
+    }
```

Comment:
I know it's not consistent with the others, but in TS you can't call code before calling `super`, so I couldn't put that in the `FireworksLLM` constructor.

Happy to get another idea on this